web/docs/setup/tune.md

```diff
@@ -333,3 +306,41 @@ vast rebuild start '#type == "suricata.flow" && #import_time < 2 weeks ago'
 ```
 
 To stop an ongoing rebuild, use the `vast rebuild stop` command.
+
+## Shutdown
+
+The `stop` command gracefully brings down a VAST server that has been started
+with the `start` command.
+
+It is also possible to send a signal `SIGINT(2)` to the `vast` process instead
+of using `vast stop`, but in only works on the same machine that runs the
+server process. We recommend using `vast stop`, as it also works over the wire.
+
+The `stop` command blocks until the server process has terminated, and returns
+a zero exit code upon success, making it suitable for use in launch system
+scripts.
```

Comment:
This is not true: The `stop` command blocks until the node actor shut down in the node, which is by definition before the process shut down. Using it in scripting via `vast stop && vast start` is not possible because of that, as there would still be a VAST process running.

(One way to fix this from our side would be to remove the PID lock as part of the node actor shut down, but that's a fix for another day.)

Reply:
Can you suggest rewording? I only copied this text to a different location and your comment doesn't is too low-level for me to derive a meaningful user-facing action.

Reply:
```suggestion
The `stop` command asks the server process to terminate gracefully, and returns
a zero exit code upon success, making it suitable for use in launch system scripts.
```